src/components/shared/Address/InvisibleCopyableMaskedAddress/InvisibleCopyableMaskedAddress.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+
+import { Address } from '~types';
+
+import MaskedAddress from '~shared/MaskedAddress';
+import InvisibleCopyableAddress from '~shared/InvisibleCopyableAddress';
+
+import styles from './InvisibleCopyableMaskedAddress.css';
+
+const displayName = 'InvisibleCopyableMaskedAddress';
+
+interface InvisibleCopyableMaskedAddressProps {
+  address: Address;
+  copyMessage?: MessageDescriptor;
+  maskedAddressStyles?: string;
+}
+
+const InvisibleCopyableMaskedAddress = ({
```

Comment:
This name is mouthful :)

Reply:
Agree, have changed to the simpler `Address`